### ISSUE TITLE
[Bug]: Multiple sensors remain in state 'unknown' after switching to UI config

### ISSUE THREAD

**Issue by Kurisutian** [mampfes/hacs_waste_collection_schedule#2517]
### I Have A Problem With:

Sensor configuration, The integration in general

### What's Your Problem

I recently switched from the yaml configuration to the UI based one. 
I took each sensor one by one from the yaml file and re-created them in the UI. It all seemed fine at first but later on I noticed something when I did receive my notification to take out the trash the evening before trash day. This collects all sensors starting with 'abfall*' and extracts the days remaining till it needs to go out. When the day shows the next day I will get notified. 
However for the Lovelace Card I have a sensor showing all entries at once following the instructions I found for that. While the card is showing all trash types correctly I noticed that the push message was missing on a bunch of them. So I checked and except one sensor each individual sensor showed "unknown" for the remaining days. 
I configured them identically and all came from the yaml file which worked before. 
Don't know why two are working and the others won't despite they are configured the same. So I presume there is something just not right. I don't really know if it is a bug or me missing something though. 

### Source (if relevant)

abfallwirtschaft_germersheim_de

### Logs

```Shell
"no relevant logs"
```

### Relevant Configuration

```YAML
Configured through UI; don't know how I can extract the config
```

### Checklist Source Error

- [X] Use the example parameters for your source (often available in the documentation) (don't forget to restart Home Assistant after changing the configuration)
- [X] Checked that the website of your service provider is still working
- [X] Tested my attributes on the service provider website (if possible)
- [X] I have tested with the latest version of the integration (master) (for HACS in the 3 dot menu of the integration click on "Redownload" and choose master as version)

### Checklist Sensor Error

- [X] Checked in the Home Assistant Calendar tab if the event names match the types names (if types argument is used)

### Required

- [X] I have searched past (closed AND opened) issues to see if this bug has already been reported, and it hasn't been.
- [X] I understand that people give their precious time for free, and thus I've done my very best to make this problem as easy as possible to investigate.

**Comment by 5ila5** [comment]
Could you check if this is the same issue as #2325 and #2506? Trailing spaces in sensor types configuration. If so, you can just open the configure page and select the sensors and remove and re-add the type and everything should be good to go

**Comment by Kurisutian** [comment]
It surely does exactly sound like that. Sorry I was unable to find that up front. I will close this one and check as mentioned there. Thanks a bunch and again sorry for the inconvinience.

**Comment by Kurisutian** [comment]
Unfortunately it did not change anything. Besides my own Value I choose (based on one of the templates) and using just value templates, the state remains unknown. Not sure what I am doing wrong here but it remained unknown.

**Comment by ckarrie** [comment]
Got the same Issue with abfallwirtschaft_germersheim_de, fixed it in #2506

**Comment by Kurisutian** [comment]
Thanks, figured it out. Had to literally type in the sensor name which I did not get in the first place. Worked after doing that. Should have gone through the instructions more carefully... ;-)
